errors/errors.go

```diff
@@ -0,0 +1,21 @@
+package errors
+
+import (
+	"os"
+)
+
+// Code exit code
+type Code int
+
+const (
+	// CodeParseRequestLength error parsing request length
+	CodeParseRequestLength Code = iota + 10
+
+	// CodeParseRequest error parsing request
+	CodeParseRequest
```

Comment:
I'd be more comfortable with explicitly declaring the values, or at least adding a comment to each line that specifies what it should be. The extension needs to use the same numbers in order to understand the errors it's receiving from the host, and being explicit here makes things easier to follow.

Reply:
Makes perfect sense, adapting.

Reply:
Thanks :-).